### PR TITLE
Fix http-cache-semantics dependency vulnerability GHSA-rc47-6667-2j5j

### PR DESCRIPTION
**Pull request by danjm**
Fixes https://github.com/advisories/GHSA-rc47-6667-2j5j

### CODE REVIEW

**Review by pedronfigueiredo — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -99,6 +99,7 @@
     "json-schema": "^0.4.0",
     "simple-get": "^4.0.1",
     "web3-provider-engine/eth-json-rpc-filters": "^5.0.0",
+    "http-cache-semantics": "^4.1.1",
```
Comment (Gudahtt) [inline package.json:102]: This is an in-range bump, so we don't need to leave a resolution here for this
```suggestion
```

Comment (Gudahtt) [reply package.json:102]: To clarify, this is a good way of bumping an indirect dependency, this is what I would do as well. Just we can remove it again after the lockfile has been updated if it's compatible with the requested ranges. This makes future updates less complicated, as conflicts with these resolutions could cause issues that would be hard to spot.

**Review by Gudahtt — DISMISSED** [review]

**Comment by Gudahtt** [comment]
Oh, another lockfile update needed, since the resolution was removed

**Review by DDDDDanica — DISMISSED** [review]

File: yarn.lock [hunk 2]
```diff
@@ -19096,10 +19096,10 @@ __metadata:
   languageName: node
   linkType: hard
 
-"http-cache-semantics@npm:^4.0.0, http-cache-semantics@npm:^4.1.0":
-  version: 4.1.0
-  resolution: "http-cache-semantics@npm:4.1.0"
-  checksum: 974de94a81c5474be07f269f9fd8383e92ebb5a448208223bfb39e172a9dbc26feff250192ecc23b9593b3f92098e010406b0f24bd4d588d631f80214648ed42
+"http-cache-semantics@npm:^4.1.1":
```
Comment (Gudahtt) [inline yarn.lock:19099]: Here is the required update:
```suggestion
"http-cache-semantics@npm:^4.0.0, http-cache-semantics@npm:^4.1.0":
```

Comment (Gudahtt) [reply yarn.lock:19099]: Hmm, GitHub isn't letting me commit this suggestion. Keeps giving an error about the diff being updated. Seems like a bug. Oh well.

Comment (danjm) [reply yarn.lock:19099]: done in latest commit

**Review by Gudahtt — APPROVED** [review]

**Review by DDDDDanica — APPROVED** [review]

**Review by brad-decker — APPROVED** [review]

**Comment by Gudahtt** [comment]
These Socket.dev warnings appear to be in error. Unsure why they were triggered. Those packages are not affected by this PR.

**Comment by brad-decker** [comment]
@SocketSecurity ignore ganache@7.7.4

**Comment by brad-decker** [comment]
@SocketSecurity ignore secp256k1@4.0.2

**Comment by brad-decker** [comment]
@SocketSecurity ignore secp256k1@4.0.3

**Comment by brad-decker** [comment]
@SocketSecurity ignore node-gyp-build@4.6.0

**Pull request merged**